### PR TITLE
[IOTDB-3865] Cluster tool show confignodes

### PR DESCRIPTION
**Pull request by RYH61**
add tool show confignodes :
图片

### CODE REVIEW

File: confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java [hunk 1]
```diff
@@ -236,6 +238,27 @@ public List<TDataNodeInfo> getRegisteredDataNodeInfoList() {
     }
     return dataNodeInfoList;
   }
+
+  public List<TConfigNodeInfo> getRegisteredConfigNodeInfoList() {
+    List<TConfigNodeInfo> configNodeInfoList = new ArrayList<>();
+    List<TConfigNodeLocation> registeredConfigNodes = this.getRegisteredConfigNodes();
+    if (registeredConfigNodes != null) {
+      registeredConfigNodes.forEach(
+          (configNodeInfo) -> {
```
Comment (CRZbulabula) [inline confignode/src/main/java/org/apache/iotdb/confignode/manager/NodeManager.java:247]:
```suggestion
          (configNodeLocation) -> {
```

**Review by CRZbulabula — COMMENTED** [review]
PTAL~

**Review by CRZbulabula — APPROVED** [review]
LGTM

**Review by qiaojialin — APPROVED** [review]

**Review by Beyyes — APPROVED** [review]

**Review by wangchao316 — APPROVED** [review]
LGTM

**Pull request merged**